json-definitions/v3/tech-record/get/psv/skeleton/index.json

```diff
@@ -470,7 +470,7 @@
         "string",
         "null"
       ],
-      "maxLength": 20
+      "maxLength": 50
```

Comment:
This one should be fine to leave at 20

Reply:
I have changed it back, don't know why it's not syncing up